### DISCUSSION TITLE
Core Updates

### DISCUSSION THREAD

**Discussion by Rohail33** [reiryuki/Miui-Core-Magisk-Module#2] [Ideas]
Even this is the initial release and just a start.. love your work man keep it up..
Just wanted to say to add support for apps like Xiaomi game turbo and Security apps... Cause they are two things a miui user is addicted to and wants in non miui roms... ( Even me) plus things like floating windows. Btw great work peeps... Looking forward for more updates

**Comment by SayaAEP** [comment]
Gameturbo is already available on miui security. But sometimes when in the game the UI doesn't appear 
https://github.com/reiryuki/Miui-Security-Center-Magisk-Module

Another reiryuki release

https://github.com/reiryuki/Miui-Gallery-Magisk-Module
https://github.com/reiryuki/Miui-Screen-Recorder-Magisk-Module
https://github.com/reiryuki/Miui-Home-Magisk-Moduleq

**Reply by SayaAEP** [reply]
(You don't really like reading descriptions so much that you don't know about the release of a security app from Reiyuki)
